reports/methodology.md

```diff
@@ -20,13 +20,19 @@ Alongside these features include:
 - author (Reddit user name)
 - date
 - post
-## Data cleaning
+## Data cleaning/transformation
+
+In order to answer the question "How has the substance use increased over the pandemic?", the feature `substance_use_total` is selected as the target feature, and the data is cleaned to focus exclusively on this feature.
+
+### Data Cleaning Steps
```

Comment:
Could you also add citations to the libraries used please?
e.g. pandas/ numpy if it was used in the script

Reply:
Yes, I will update the citations.